src/features/content-answers/ContentAnswers.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+import ContentAnswersOpenButton from './ContentAnswersOpenButton';
+
+const ContentAnswers = () => {
+    const [openContentAnswersModal, setOpenContentAnswersModal] = useState(false);
```

Comment:
nit: this naming would be better for the boolean
```suggestion
    const [isModalOpen, setIsModalOpen] = useState(false);
```